umm-spec-lib/src/cmr/umm_spec/test/iso19115_expected_conversion.clj

```diff
@@ -321,6 +322,58 @@
         role (:Roles data-center)]
    (assoc data-center :Roles [role]))))
 
+(defn- update-contact-persons-from-collection-citation
+  "CollectionCitation introduces additional contact persons from creator, editor and publisher.
+   They need to be added to the ContactPersons in the expected umm after converting the umm
+   to the xml and back to umm.  Returns the updated ContactPersons"
+  [contact-persons collection-citation]
+  (let [creator (:Creator collection-citation)
+        editor (:Editor collection-citation)
+        publisher (:Publisher collection-citation)
+        creator-contact-person (when creator
```

Comment:
:)  I agree. I was just so frustrated with all the errors and was happy to just get it "working"...  I know I have a lot of refactoring to do...  Thanks.